x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/external/request/elastic/ElasticInferenceServiceSparseEmbeddingsRequestEntity.java

```diff
@@ -34,8 +40,15 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         }
 
         builder.endArray();
+
+        // optional field
+        if ((usageContext == ElasticInferenceServiceUsageContext.UNSPECIFIED) == false) {
```

Comment:
This is the entity that gets marshalled into JSON in the outgoing request, yes? If we're omitting `"usage_context": "unspecified"`, then there's a discrepancy with the [spec](https://github.com/elastic/inference-task-api-spec/pull/28/files#diff-85ef6f86318e88608aad63946765f58487d02ea239c75748bebb192cbd285419R527) - consider updating it.

Reply:
Good point, I'll think about it after FF, if we simply want to update the spec (it says that it's optional though...maybe we should treat a missing `usage_context` simply as `unspecified` on EIS?)